src/libraries/System.IO.FileSystem/tests/Enumeration/SpecialDirectoryTests.cs

```diff
@@ -60,7 +60,14 @@ public void SkippingHiddenFiles()
         {
             // Files that begin with periods are considered hidden on Unix
             string[] paths = GetNames(TestDirectory, new EnumerationOptions { ReturnSpecialDirectories = true, AttributesToSkip = 0 });
-            Assert.Contains(".", paths);
+
+            if (!PlatformDetection.IsWindows10Version22000OrGreater)
```

Comment:
Why disabling for Windows 10? Is it failing consistently there as well?

Reply:
This is the version number that Windows 11 reports.

Reply:
Oh got it, thanks for clarifying.

Reply:
Yes, we could probably rename this as IsWindows11 but that's what it is currently.